src/System.Runtime/ref/System.Runtime.Manual.cs

```diff
@@ -33,6 +33,7 @@ public partial struct Single
         // We need to add this into the manual ref assembly to preserve it because the
         // implementation doesn't have any reference field, hence GenApi will not emit it.
         private object _dummy;
+        // Placing the value type field in the manual ref as well to avoid the error CS0282: There is no defined ordering between fields in multiple declarations of partial struct 'TypedReference'.
         private int _dummyPrimitive;
```

Comment:
Since the value type field **does** get generated by GenAPI (for the `IntPtr` fields), wanted to add an explicit comment to highlight why we want to keep that in the manual file instead.

https://github.com/dotnet/coreclr/blob/10bf24055157e962b50c6bb25865f69fb055cd98/src/System.Private.CoreLib/src/System/TypedReference.cs#L19-L22

cc @safern since you added this in your change here: https://github.com/dotnet/corefx/pull/37737